hi-ml/src/health_ml/runner.py

```diff
@@ -245,7 +245,13 @@ def after_submission_hook(azure_run: Run) -> None:
         temp_conda_file: Optional[Path] = None
         try:
             if self.experiment_config.cluster:
-                env_file = choose_conda_env_file(env_file=self.experiment_config.conda_env)
+                # Enable spawned processes to find relative env file regardless of CWD
+                conda_env_file = self.experiment_config.conda_env
+                if conda_env_file is not None:
+                    if str(self.project_root) not in str(conda_env_file):
+                        conda_env_file = self.project_root / conda_env_file
+
+                env_file = choose_conda_env_file(env_file=conda_env_file)
```

Comment:
I think this can be done simpler. We need to prevent lines 248 to be executed in AzureML altogether. Modify line 247 to be `if self.experiment_config.cluster and not is_running_in_azureml():`